R/mixture_identification.R

```diff
@@ -22,10 +23,13 @@
 #'   \item wear-out failures.
 #' }
 #' In order to reduce the risk of overestimation as well as being able to consider
-#' the main types of failures, a maximum of three subgroups can be obtained.
+#' the main types of failures, a maximum of three subgroups (\code{k = 3}) is recommended.
 #'
 #' @inheritParams rank_regression.wt_cdf_estimation
 #'
+#' @param k Number of mixture components. If the data should be split in an automated
+#'   fashion \code{k} must be set to \code{NULL}. The argument \code{fix.psi}
```

Comment:
"," if if-clause starts a sentence, no "," it it ends a sentence. I had to look this up as well.